datasm/dataset.py

```diff
@@ -94,7 +94,7 @@ def __init__(
 
         self.versions = versions
 
-        log_message("debug", f"init: splitting to obtain facets: dsid = {self.dataset_id}")
+        log_message("debug", f"dataset.py init: splitting to obtain facets: dsid = {self.dataset_id}")
```

Comment:
```suggestion
        log_message("debug", f"{__name__} init: splitting to obtain facets: dsid = {self.dataset_id}")
```
You can use `__name__` to get the name of the module instead of manually typing it out

Reply:
@tomvothecoder Excellent.  I'll seek to replace these.

Is there a corresponding `{__func__}` that will obtain the name of the function in which a message is generated?